app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -260,7 +276,12 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
       return window.location.hostname
     },
 
-    removeCrossDomainParams: function (url) {
+    getProtocol: function () {
+      return window.location.protocol
+    },
+
+    removeCrossDomainParams: function (clickData) {
```

Comment:
Same comment here about returning values as for the `populateLinkDomain` function.

Reply:
See here https://github.com/alphagov/govuk_publishing_components/pull/3004#discussion_r990078024